app/assets/javascripts/application.js

```diff
@@ -1,3 +1,6 @@
+import 'app/autoLogout';
+import 'app/hms-formatter';
+import 'app/countdownTimer';
```

Comment:
yep, you are right, I payed attention for the formatter but named on autopilot for the other two